plugins/wasi_nn/wasinn_ggml.h

```diff
@@ -25,16 +25,20 @@ struct Graph {
   llama_model *LlamaModel = nullptr;
   std::string ModelFilePath;
   llama_context *LlamaContext = nullptr;
+  struct clip_ctx *ClipContext = nullptr;
   // Plugin parameters:
   bool EnableLog = false;
   bool EnableDebugLog = false;
   bool StreamStdout = false;
   bool Embedding = false;
+  int32_t EmbdNormalize =
+      2; // -1=none, 0=max absolute int16, 1=taxicab, 2=euclidean, >2=p-norm
```

Comment:
Should we use `enum class` for this?

Reply:
Just add the `EmbdNormalizeType` for this.